### PR TITLE
Create simulations_info.md

### PR DESCRIPTION
**Pull request by manodeep**
@pelahi Will you please double-check the numbers. 

I assume it is okay for this list to be public...

Related to #4. Not ready to be merged yet.

### CODE REVIEW

**Comment by pelahi** [comment]
@manodeep Updated the info but haven't listed cadence yet. What would you like listed?

**Comment by lspitler** [comment]
@manodeep @pelahi maybe add `h` assumed to the table

**Comment by manodeep** [comment]
@lspitler The units listed in the columns show the little-h. Do you mean to incorporate the units along with the numbers?

**Comment by manodeep** [comment]
@pelahi What I understood from Chris yesterday was that the 3800 has been replaced by a sim > 4000^3. And he may have mentioned another 8192^3 simulation as well. For snapshot cadence, I figure simply quoting the total number of snapshots might be sufficient for this table.

**Comment by lspitler** [comment]
@manodeep I meant provide the little-h that was assumed in the table so its self-contained.

**Comment by manodeep** [comment]
@pelahi Do you mind posting the updated numbers here? We can merge in this pull request

**Comment by pelahi** [comment]
Done

**Comment by manodeep** [comment]
Thanks @pelahi 

@lspitler Do you think this PR is fine to merge in and resolve #4 ?

File: simulations_info.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# List of Genesis simulations
+
+Spatial resolution is particle gravitional softening; any scale less than ~3x the softening is not reliable. Halo structural properties are only reliable when > 200 particles are present in the halo. 
+
+| Simulation Type | Box side length (Mpc/h) |  Particle Number | Particle mass (M⊙/h) | Minimum redshift current (planned) | Spatial Resolution (kpc/h) | Snapshot cadence |
+| --------------- | ----------------------- | ---------------- | -------------------- | ---------------------------------- | -------------------------- | ---------------- |
+| DM-only         | 35                      | 2650^3           | 2.0e5                |    0.0  (5.0)                      | ~0.25
```
Comment (lspitler) [inline simulations_info.md]: @manodeep @pelahi should the Minimum redshifts for the DM-only super zoom be: current=5 and planned=0 ?

**Review by lspitler — APPROVED** [review]
All good, fine to merge after addressing one quick question.

File: simulations_info.md (hunk 1 above)
Comment (pelahi) [reply simulations_info.md]: Should be 5.0 (5.0) for this simulation.

**Pull request merged**